src/backend/environment.ts

```diff
@@ -15,49 +15,61 @@ const isDev = () => {
 // Se på disse url-ene. Her lenker dev og env til hele lenken, mens den siste har bare port 8082 og sender med resten i api-service
 
 export interface EnvironmentProps {
-  loginService: string;
+  wonderwallUrl: string;
   apiUrl: string;
+  apiProxyUrl: string;
+  dokumentProxyUrl: string;
   dokumentUrl: string;
-  mergeDokumentUrl: string;
   port: number;
   dekoratørEnv: 'dev' | 'prod';
+  oauthCallbackUri: string;
 }
 
 const getEnv = (): EnvironmentProps => {
   if (isProd()) {
     return {
-      loginService: 'https://loginservice.nav.no/login?',
+      wonderwallUrl:
+        'https://familie.nav.no/familie/alene-med-barn/soknad/oauth2/login?redirect=',
```

Comment:
Er det riktig med familie som subdomain her?

Reply:
Det har du nok rett i at det ikke er. Dette var kopiert fra familie-ef-soknad, så da har du funnet to feil på en gang hehe :D

Reply:
💯